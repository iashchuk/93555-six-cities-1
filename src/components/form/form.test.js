import React from "react";
import Form from "./form.jsx";
import renderer from "react-test-renderer";

describe(`Form`, () => {
  it(`renders correctly`, () => {
    const component = renderer.create(<Form />).toJSON();

    expect(component).toMatchSnapshot();
  });
});
