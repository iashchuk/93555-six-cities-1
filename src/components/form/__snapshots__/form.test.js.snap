// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form renders correctly 1`] = `
<form
  action="#"
  className="places__sorting"
  method="get"
>
  <span
    className="places__sorting-caption"
  >
    Sort by
  </span>
  <span
    className="places__sorting-type"
    tabIndex="0"
  >
    Popular
    <svg
      className="places__sorting-arrow"
      height="4"
      width="7"
    >
      <use
        xlinkHref="#icon-arrow-select"
      />
    </svg>
  </span>
  <ul
    className="places__options places__options--custom places__options--opened"
  >
    <li
      className="places__option places__option--active"
      tabIndex="0"
    >
      Popular
    </li>
    <li
      className="places__option"
      tabIndex="0"
    >
      Price: low to high
    </li>
    <li
      className="places__option"
      tabIndex="0"
    >
      Price: high to low
    </li>
    <li
      className="places__option"
      tabIndex="0"
    >
      Top rated first
    </li>
  </ul>
  <select
    className="places__sorting-type"
    id="places-sorting"
  >
    <option
      className="places__option"
      value="popular"
    >
      Popular
    </option>
    <option
      className="places__option"
      value="to-high"
    >
      Price: low to high
    </option>
    <option
      className="places__option"
      value="to-low"
    >
      Price: high to low
    </option>
    <option
      className="places__option"
      value="top-rated"
    >
      Top rated first
    </option>
  </select>
</form>
`;
